test/fw/ptl/utils/pbs_crayutils.py

```diff
@@ -0,0 +1,114 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2017 Altair Engineering, Inc.
```

Comment:
Shouldn't 2017 be 2018?

Reply:
2017 for now, We usually change the year throughout the project in one shot.

Reply:
I have updated the text per changes in PP-1174.